public/openapi3.json

```diff
@@ -0,0 +1,9983 @@
+{
```

Comment:
We have to let prettier ignore the `public/openapi3.json` same as it's done [here](53170)

Reply:
Fixed in aefef610fa592330337125ac5e1ff9ccaebbb2ed